MentionmeSwift/Classes/Requests/MentionmeReferrerEnrollmentRequest.swift

```diff
@@ -10,8 +10,13 @@ import Foundation
 
 public class MentionmeReferrerEnrollmentRequest: MentionmeRequest {
 
-  public override init() {
+  override init() {
     super.init()
+  }
+
+  public convenience init(mentionmeCustomerParameters: MentionmeCustomerParameters) {
+
+    self.init()
```

Comment:
Am I right in saying this change brings the code inline with the other Request objects?